rnd/autogpt_server/autogpt_server/data/block.py

```diff
@@ -13,6 +14,25 @@
 BlockOutput = Generator[BlockData, None, None]
 
 
+class BlockCategory(Enum):
+    class Obj(BaseModel):
+        category: str
+        description: str
+
+        @classmethod
+        def get(cls, category: str, description: str) -> member:
+            obj = cls(category=category, description=description)
+            return member(obj)
+
+    llm = Obj.get("llm", "Block that uses the Large Language Model to perform a task.")
+    http = Obj.get("http", "Block that executes a network request through HTTP.")
+    social = Obj.get("social", "Block that interacts with social media platforms.")
+    text = Obj.get("text", "Block that processes text data.")
+    file = Obj.get("file", "Block that reads or writes files.")
+    web = Obj.get("web", "Block that interacts with web pages.")
+    basic = Obj.get("basic", "Basic blocks that perform basi operations.")
```

Comment:
Convention for Python enum values is SCREAMING_SNAKE_CASE

Reply:
Updated